docs/reference/ml/df-analytics/apis/get-dfanalytics-stats.asciidoc

```diff
@@ -62,11 +62,439 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=size]
+//Begin reg_hyperparameters
+`hyperparameters`::::
+(object)
+An object containing the parameters of the {reganalysis}.
++
+.Properties of `hyperparameters`
+[%collapsible%open]
+=======
+`alpha`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-alpha]
+
+`downsample_factor`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-downsample-factor]
+
+`eta`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-eta]
+
+`eta_growth_rate_per_tree`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-eta-growth]
+
+`feature_bag_fraction`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-feature-bag-fraction]
+
+`gamma`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-gamma]
+
+`lambda`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-lambda]
+
+`max_attempts_to_add_tree`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-max-attempts]
+
+`max_optimization_rounds_per_hyperparameter`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-max-optimization-rounds]
+
+`max_trees`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-max-trees]
+
+`num_folds`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-num-folds]
+
+`num_splits_per_feature`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-num-splits]
+
+`soft_tree_depth_limit`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-soft-limit]
+
+`soft_tree_depth_tolerance`::::
+(double)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-soft-tolerance]
+=======
+//End reg_hyperparameters
+
+`iteration`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-iteration]
+
+`timestamp`::::
+(date)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-timestamp]
+
+//Begin reg_timing_stats
+`timing_stats`::::
+(object)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-timing-stats]
++
+.Propertis of `timing_stats`
+[%collapsible%open]
+=======
+`elapsed_time`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-timing-stats-elapsed]
+
+`iteration_time`::::
+(integer)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-timing-stats-iteration]
+=======
+//End reg_timing_stats
+
+//Begin reg_validation_loss
+`validation_loss`::::
+(object)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-validation-loss]
++
+.Properties of `validation_loss`
+[%collapsible%open]
+=======
+`fold_values`::::
+(array of strings)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-validation-loss-fold]
+
+`loss_type`::::
+(string)
+include::{docdir}/ml/ml-shared.asciidoc[tag=dfas-validation-loss-type]
+=======
+//End reg_validation_loss
+======
+//End regression_stats
+=====
+//End analysis_stats
+
+`assignment_explanation`:::
+(string)
+For running jobs only, contains messages relating to the selection of a node to 
+run the job.
+
+//Begin data_counts
+`data_counts`:::
+(object)
+An object containing statistical data about the documents in the analysis.
```

Comment:
It's confusing since it is the same text as for `analysis_stats`. I guess @dimitris-athanasiou can advise better on what to write here.

Reply:
Thanks, I've drafted alternative text.